### ISSUE TITLE
More convenient access to mink functions

### ISSUE THREAD

**Issue by romanzks** [soulcodex/laravel-behat#2]
Instead of this
```php
    /**
     * @Given I send a request to :url
     */
    public function iSendARequestTo(string $url): void
    {
        $session = $this->session(); // The current mink session
        $session->visit($url); // Perform an action using the mink session 
    }
```
it would be great to have it
```php
    /**
     * @Given I send a request to :url
     */
    public function iSendARequestTo(string $url): void
    {
        $this->visit($url); // Perform an action using the mink session 
    }
```
this approach is used in other similar packages. this saves us from having to get the current session at the beginning of each function. this is usually implemented through inheritance.

**Comment by soulcodex** [comment]
Hi @romanzks 

Its now available the version 1.0.4 on packagist, let me know your thought about this new features, have a nice week.

Best regards

**Comment by romanzks** [comment]
Thanks for update, but unfortunately I'm still can't reach Mink functions
`vendor/friends-of-behat/mink-extension/src/Behat/MinkExtension/Context/MinkContext.php`

```php
use Soulcodex\Behat\Addon\Context;

final class FeatureContext extends Context
{
    /** @Given I am logged as admin */
    public function givenIAmLoggedAsAdmin()
    {
        $this->visit("/login");
        $this->fillField("email", 'admin@admin.com');
        $this->fillField("password", 'password');
        $this->pressButton("Login");
    }
}
```

```bash
Scenario: Log in as admin                              # features/auth.feature:23
    Given I am logged as admin                           # FeatureContext::givenIAmLoggedAsAdmin()
      Fatal error: Call to undefined method PHPUnit\Framework\Assert::visit() (Behat\Testwork\Call\Exception\FatalThrowableError)
```
